source/codegen/service_helpers.py

```diff
@@ -75,8 +75,6 @@ def create_args(parameters):
             result = f'{result}({type_without_brackets}*){parameter_name}.data(), '
         elif parameter['type'] in {"ViBoolean[]", "ViSession[]", "ViInt16[]"}:
             result = f'{result}{parameter_name}.data(), '
-        elif parameter.get('is_size_param', False) and is_twist_mechanism:
```

Comment:
Note: I thought this special case in `create_args` was pretty confusing and it also broke this change. This is replaced with the new bit of mako:


```
% if not is_in_out_twist:
        auto ${size_param_name} = ${twist_param_name};
% endif
```
Which allows create_args to treat the size_param as a regular param, instead of special-case swapping out the twist param.